### DISCUSSION TITLE
Can I modify or fix navmesh data by hand after it's generated?

### DISCUSSION THREAD

**Discussion by gy373499700** [recastnavigation/recastnavigation#563] [Q&A] [answered]
navmesh 生成后，如果数据不合理，我希望手动修复一下，如何进行？

Translation: 
> After navmesh is generated, if the data is not reasonable, I want to fix it manually, how to do it?

**Comment by grahamboree** [comment, marked as answer]
The navmesh data contained in `dtNavMesh` contains a list of nav tiles which each contain polygon and vertex information (amongst other things) and can be edited however you see fit.
